fuzzers/Makefile

```diff
@@ -8,6 +8,8 @@ FUZZONLY=N
 # Allow building without it
 BITONLY=N
 
+all:
```

Comment:
Also need a .PHONY for all?

Reply:
I don't think so. In my tests this works fine without PHONY

Reply:
Please add it, see https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html

The reason it is working is there is not a file named all.

Try this:
```
make clean
touch all
make all
```
It should probably do nothing

Reply:
Though I guess all has dependencies, so it might work out.

Reply:
actually all the fuzzer's run.ok files are `all` target dependencies.

Reply:
You probably want to make the `all` target the default? 
```makefile
.DEFAULT_GOAL := all
```